ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -949,26 +965,34 @@ static GstCaps *
 _ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR * self)
 {
   const GstTensorInfo *spec;
-  gchar *tensor;
   GstCaps *retval;
 
-  /** @bug the retrieved spec is not valid...? */
-  spec = g_hash_table_lookup (tizensensors, GINT_TO_POINTER (self->type));
+  spec = self->src_spec;
 
   if (FALSE == self->configured || SENSOR_ALL == self->type || NULL == spec) {
-    tensor = g_strdup_printf ("other/tensor; other/tensors, num_tensors=1");
+    retval = gst_caps_from_string (GST_TENSOR_CAP_DEFAULT "; "
+      GST_TENSORS_CAP_DEFAULT);
```

Comment:
```GST_TENSORS_CAP_DEFAULT``` = ```other/tensors, num_tensors = (int) (1, 16], framerate = ...```. It makes pipeline initialization slower than ```other/tensors, num_tensors = 1``` as long as we can sure that the number of tensors is always 1.

Reply:
Right, thanks! I updated the caps with `GST_TENSORS_CAP_WITH_NUM ("1")`. Please check!